skycultures/arabic/constellation_names.eng.fab

```diff
@@ -0,0 +1,138 @@
+0100 "الفرقدان" _("The Two Oryx Calves")
+0200 "فأس الرحى" _("The Axe Of The Grindstone")
+# TRANSLATORS: Na'sh is a proper name meaning bier
+0300 "بنات نعش" _("Daughters Of Na'sh")
+0400 "الظباء وأولاد الظباء" _("The Gazelles And Thier Youngsters")
+0410 "القفزة الأولى" _("The First Leap")
+0420 "القفزة الثانية" _("The Second Leap")
+0430 "القفزة الثالثة" _("The Third Leap")
+0501 "الفارطان" _("The Two Preceding To The Water")
+0500 "الحوض" _("The Pool")
+0600 "الذئبان" _("The Two Wolves")
+0601 "أظفار الذئب" _("The Wolf Claws")
+0700 "العوائذ" _("The Mother Camels")
+0800 "الأثافي" _("The Three Stone Support")
+0900 "القدر" _("The Pot")
+1000 "الراعي الشمالي والشاء" _("The Shepherd And The Sheep")
+1001 "كوكبا الفرق" _("The Two Separated Stars")
+1100 "ذو السلاح" _("The Armed One")
+1101 "الرمح" _("The Spear")
+1200 "السماك الأعزل" _("The Unarmed One")
+1300 "الضباع وأولاد الضباع" _("The Hyenas And Their Youngsters")
+#3 page
+1400 "قصعة المساكين" _("The Bowl Of The Poor")
+1500 "الروضة والأغنام" _("The Desert Garden And The Goats")
+1501 "النسق الشآمي" _("The Northern Line")
+1502 "النسق اليماني" _("اthe Southern Line")
+1600 "النسر الواقع" _("The Landing Eagle")
+1700 "النسر الطائر" _("The Flying Eagle")
+1800 "الفوارس" _("The Knights")
+1900 "الناقة" _("The She-Camel")
+1901 "رأس الناقة" _("Head Of Tthe She-Came")
+1902 "عنق الناقة" _("Neck Of Tthe She-Came")
+1903 "يدا الناقة" _("Legs Of Tthe She-Came")
+#The name is left blank intentionally, label of 2604 Al-Thurayya is sufficient. This constellation is to show the complete Al-Thurayya image with its two hands.
+#2000 "الثريا " _("Al-Thurayya")
+2000 " " _("")

```

Comment:
Why it empty?

Reply:
This constellation has a small part (2604) with the same name and the label is better placed on the screen.
https://github.com/kajaji/stellarium/blob/c0882bae2304b2662464ebc6335eeccb82222717/skycultures/arabic/constellation_names.eng.fab#L65
The bigger part (2000) label would go inside another constellation and could cause confusion.
![Thurayya](https://user-images.githubusercontent.com/34771982/143844925-b7a045e4-9929-440b-bf47-89482cd62f77.jpg)